packages/trader/src/AppV2/Components/Chip/chip.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { LabelPairedChevronDownSmRegularIcon } from '@deriv/quill-icons';
+import './chip.scss';
+import clsx from 'clsx';
+import { Text } from '@deriv-com/quill-ui';
+import { TRegularSizes } from '@deriv-com/quill-ui/dist/types';
+
+type BaseChipProps = Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'label'> & {
+    label?: React.ReactNode;
+    disabled?: boolean;
+    isDropdownOpen?: boolean;
+    dropdown?: boolean;
+    selected?: boolean;
+    size?: TRegularSizes;
+    onClick?: () => void;
+};
+
+const Chip = React.forwardRef<HTMLButtonElement, BaseChipProps>(
```

Comment:
Can we use the Chip component from quill instead?
Why do we need to create one here?

Reply:
Because currently in quill chip exists only together with dropdown (i mean list of options), there is no pure exportable chip itself. So I took everything I need from quill's code (structure, css) a create small component on our package. I'll try to create PR into Quill

Reply:
I mean I'll try to create a PR to make 'base' chip exportable although it has a lot of unused for us logic

Reply:
Sure. Thanks @kate-deriv. 
We can inform the Quill team to create the base `chip` for us.